### PR TITLE
chore: update editorconfig and fix indent in views

### PR DESCRIPTION
**Pull request by kenjis**
Supersedes #6010

**Description**
Indent the View files with 4 spaces.
The only one exception is `system/Debug/Toolbar/Views/toolbar.css`.
It is generated by scss and it uses space 2 indentation.

- update editorconfig
- fix indentation in View files

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: app/Views/errors/html/debug.css [hunk 1]
```diff
@@ -1,197 +1,197 @@
 :root {
-	--main-bg-color: #fff;
-	--main-text-color: #555;
-	--dark-text-color: #222;
-	--light-text-color: #c7c7c7;
-	--brand-primary-color: #E06E3F;
-	--light-bg-color: #ededee;
-	--dark-bg-color: #404040;
+    --main-bg-color: #fff;
+    --main-text-color: #555;
```
Comment (paulbalandan) [inline app/Views/errors/html/debug.css:3]: Are these diffs generated by Dart SASS?

Comment (kenjis) [reply app/Views/errors/html/debug.css:3]: Oh, it seems my mistake.

Comment (kenjis) [reply app/Views/errors/html/debug.css:3]: No, it is not generated by SASS.
I replaced tabs with spaces.

Comment (paulbalandan) [reply app/Views/errors/html/debug.css:3]: Ok. It seems kinda weird that they look the same from my PC.

image

**Review by paulbalandan — APPROVED** [review]
This is fine with me but let's have others take a look.

**Review by MGatner — APPROVED** [review]

**Comment by MGatner** [comment]
Coverage decreased 6% 😭

**Comment by kenjis** [comment]
I don't think this PR changes code coverage.

I saw https://coveralls.io/builds/49510047 but I don't get where the decreasing occurs.

**Pull request merged**